mountpoint-s3/tests/reftests/reference.rs

```diff
@@ -141,7 +141,11 @@ impl Reference {
     pub fn new(remote_keys: Vec<(String, MockObject)>) -> Self {
         let local_files = vec![];
         let local_directories = vec![];
-        let materialized = build_reference(remote_keys.iter().map(|(k, o)| (k, o)));
+        let remote_key_iter = remote_keys.iter().map(|(k, o): &(String, MockObject)| {
+            let tuple_of_refs: (&String, &MockObject) = (k, o);
+            tuple_of_refs
+        });
+        let materialized = build_reference(remote_key_iter);
```

Comment:
It's been fixed already in clippy, but while we wait for the next release, maybe we could just simply have:

```suggestion
        let materialized = build_reference(remote_keys.iter().map(|(k, o): &(_, _)| (k, o)));
```